Vaibhav Shukla

```diff
@@ -0,0 +1 @@
+
```

Comment:
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Semantic </title>
  <style>
    body {
      font-family: Arial, sans-serif;
      margin: 0;
      padding: 0;
      background-color:purple;
    }

    header {
      background-color: grey;
      color: white;
      padding: 20px;
      text-align: center;
    }

    nav {
      background-color: blue;
      padding: 10px;
      text-align: center;
    }

    nav a {
      color:pink;
      text-decoration: none;
      padding: 10px;
      margin: 0 15px;
    }

    nav a:hover {
      background-color:brown;
    }

    main {
      margin: 20px;
      display: flex;
      justify-content: space-between;
    }

    article {
      width: 65%;
    }

    article h2 {
      font-size: 24px;
    }

    article p {
      line-height: 1.6;
      font-size: 18px;
    }



    footer {
      background-color:grey;
      color: white;
      text-align: center;
      padding: 20px;
      margin-top: 20px;
    }
</style>
</head>
<body>
  <header>
    <h1>My Semantic Webpage</h1>
    <p>Interesting articles</p>
  </header>
  <nav>
    <a href="#">Home</a>
    <a href="#">Articles</a>
    <a href="#">About</a>
    <a href="#">Contact</a>
  </nav>
  <main>
    <article>
      <h2>Understanding the Importance of Web Semantics</h2>
      <p>
        Web semantics is about using HTML tags to convey meaning, making content more accessible and enhancing SEO. Semantic elements allow browsers and developers to understand the context of the content better, which can improve page performance and search rankings.
      </p>
      <p>
        Semantic elements are not only important for readability but also for ensuring that web pages are more accessible for people with disabilities. For example, assistive technologies like screen readers can interpret semantic HTML more accurately than non-semantic tags, making the content more inclusive.
      </p>
    </article>
  
  </main>
  <footer>
    <p>© 2024 My Semantic Webpage. All rights reserved.</p>
    <p>Follow us on <a href="#" style="color: white;">Twitter</a> and <a href="#" style="color: white;">Facebook</a>.</p>
  </footer>

</body>
</html>